src/UI/Art/SpriteArtist/SpriteOptions.java

```diff
@@ -81,7 +77,7 @@ private static void initialize()
     // Calculate the size of the longest item panel needed.
     for( int i = 0; i < allOptions.length; ++i )
     {
-      String[] allItems = allOptions[i].optionList;
+      String[] allItems = (String[])allOptions[i].optionList;
```

Comment:
I'm not a super fan of this cast.

Reply:
I normally try to avoid casts, yeah. I'm considering what it would take to make `GameOption` generic, and hopefully avoid the need for casting everywhere.

Reply:
Done in new PR.